testing/widgets/common/DialogsAndNotificationsOverview.test.js

```diff
@@ -16,7 +16,7 @@ runManualTest('Common', 'DialogsAndNotificationsOverview', ['jQuery', 'React', '
   test('DialogsAndNotificationsOverview', async (t) => {
     const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
 
-    await t.click($(`.${IMAGES_CONTAINER_CLASS} .${ITEM_CONTENT_CLASS} img`));
+    await t.click($(`.${IMAGES_CONTAINER_CLASS} .${ITEM_CONTENT_CLASS}`));
```

Comment:
Why do we need change this?

Reply:
Yes, it's unsufficient selector for testcafe. This change allow prevent warning appearence and blinking of the test.